ui/src/views/Rules/Rule/index.jsx

```diff
@@ -395,13 +400,73 @@ function Rule({ isNewRule, user, ...props }) {
     return `Update Rule ${ruleId}${rule.alias ? ` (${rule.alias})` : ''}`;
   };
 
+  const [requiredSignoffs, fetchRequiredSignoffs] =
+  useAction(getRequiredSignoffs);
+
+  useEffect(() => {
+    let myPromise = new Promise((res) => {
+      res(fetchRequiredSignoffs(OBJECT_NAMES.PRODUCT_REQUIRED_SIGNOFF));
+    });
+    myPromise.then((rs) => {
+      setAllrequiredSignoffs(rs.data.data.required_signoffs);
+    });
+  }, []);
+
+  // SignOff style
+  const signoffStyle = {
+    div: {
+      display: "flex",
+      flexFlow: "row wrap",
+      width: "100%",
+      margin: "8px 0 8px 0",
+      alignItems: "center",
+    },
+    p1: {
+      marginRight: "8px",
+      color: "gray",
+    },
+    p2: {
+      color: "black",
+    },
+  };
+
   return (
     <Dashboard title={getTitle()}>
-      {isLoading && <Spinner loading />}
-      {error && <ErrorPanel fixed error={error} />}
-      {!isLoading && (
-        <Fragment>
-          <div className={classes.scheduleDiv}>
+    {isLoading && <Spinner loading />}
+    {error && <ErrorPanel fixed error={error} />}
+    {!isLoading && (
+      /*
+      THE SIGN OFF DETAIL SHOULD GO JUST UNDERNEATH THE OPENING FRAGMENT
+      It should be in the form : 
+      "SignOffs Required : 1 signoff from a releng user" 
+      */
+      <Fragment>
+        {/* SHOWS REQUIRED SIGNOFFS */}
+        <div style={signoffStyle.div}>
+          <Typography component="p" variant="body2" style={signoffStyle.p1}>Required Signoff(s) :</Typography>
+          {allRequiredSignOffs.map((reqSignOff, index, all) => {
+            if (
+              rule.product === reqSignOff.product &&
+              rule.channel === `${reqSignOff.channel}*`
+            ) {
+              return (
+                <Typography component="p" variant="body2" key={index} style={signoffStyle.p2}>{`${
+                  reqSignOff.signoffs_required
+                } member${reqSignOff.count > 1 ? "s" : ""} of ${
+                  reqSignOff.role
+                }${all[index + 1] ? "," : ""}`}</Typography>
+              );
+            } else {
+              return (
+                <Typography component="p" variant="body2" key={index} style={signoffStyle.p2}>
+                  No signoffs provided for this product in local development
```

Comment:
I'm going to let @gabrielBusta do the full review of this, but I do want to make sure that we don't hardcode a string like this. Even in production there are some products/channels that do not require signoffs. A simple string of "None" would be just fine here.

Reply:
Noted. Changed this. I'm still trying to fix lint errors in my code though. `yarn lint --fix` is giving me errors still. I hope I fix this early enough

Reply:
Done fixing eslint errors now :)

Reply:
Just added a fix for the release channel . Meant to be "release" and not "release*" :)

Reply:
In addition, this also works when creating rules, as soon as you select it's product name and channel, it'll immediately show you the required signoffs right there! :)